### PR TITLE
Upgrade Second Life Viewer.app to 3.7.28.300918, add stanzas

### PR DESCRIPTION
**Pull request by mjgardner**
License here: http://wiki.secondlife.com/wiki/Linden_Lab_Official:Second_Life_Viewer_Licensing_Program

System requirements here: https://secondlife.com/support/system-requirements/

### CODE REVIEW

**Comment by vitorgalvao** [comment]
There’s trailing whitespace that needs to be removed.

File: Casks/second-life-viewer.rb [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
   homepage 'http://secondlife.com/'
-  license :unknown    # todo: change license and remove this comment; ':unknown' is a machine-generated placeholder
+  license :oss
+  tags :vendor => 'Linden Lab'
 
   app 'Second Life Viewer.app'
+  
+  depends_on :macos => '>= :lion'
+  
+  zap :delete => [  '~/Library/Application Support/SecondLife',
```
Comment (vitorgalvao) [inline Casks/second-life-viewer.rb]: Should be

```ruby
zap :delete => [
                '~/Library/Application Support/SecondLife',
                '~/Library/Caches/SecondLife',
               ]
```

to follow our style.

File: Casks/second-life-viewer.rb [hunk 2]
```diff
@@ -1,11 +1,18 @@
...
-  version '3.6.13.284995'
-  sha256 '8aa1bc39077452c3006390d4a888ca4113c087e8cdc78f5008dc85091015627d'
+  version '3.7.28.300918'
+  sha256 '2d6fa6c60a3bfbd5ed6b08b5455ed6b0d4c7820b7c5f60b50f9a20be371e91ef'
 
   url "http://download.cloud.secondlife.com/Viewer_3/Second_Life_#{version.gsub('.','_')}_i386.dmg"
   name 'Second Life Viewer'
   homepage 'http://secondlife.com/'
-  license :unknown    # todo: change license and remove this comment; ':unknown' is a machine-generated placeholder
+  license :oss
```
Comment (vitorgalvao) [inline Casks/second-life-viewer.rb]: Ins’t there a specific license?

Comment (mjgardner) [inline Casks/second-life-viewer.rb]: According to the [vendor's wiki page on licensing](http://wiki.secondlife.com/wiki/Linden_Lab_Official:Second_Life_Viewer_Licensing_Program) it is LGPL 2.1. I don't see a symbol for LGPL available, just GPL and AGPL.

Comment (vitorgalvao) [inline Casks/second-life-viewer.rb]: No matter, you can set it to `:gpl`. `license` is just a general guideline.

**Pull request closed without merging**